mod_cluster/extension/src/main/resources/schema/jboss-as-mod-cluster_4_0.xsd

```diff
@@ -301,17 +301,25 @@
                 </xs:annotation>
             </xs:element>
         </xs:choice>
+        <xs:attribute name="decay" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>
+                    The factor by which a historic load values should degrade in significance.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
         <xs:attribute name="history" type="xs:string">
             <xs:annotation>
                 <xs:documentation>
                     The number of historic (previous) load values to consider in the load balance factor computation.
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="decay" type="xs:string">
+        <xs:attribute name="initial-load" type="xs:string">
             <xs:annotation>
                 <xs:documentation>
-                    The factor by which a historic load values should degrade in significance.
+                    Initial load within the range [0..100] to prepopulate historical values with. Used to gradually drive load
```

Comment:
"Initial load within the range [0..100] with which to prepopulate historical values."

Reply:
Fixed. One should never end a sentence preposition with.